composer.json

```diff
@@ -20,8 +20,11 @@
         "symfony/security-core": "^3.4|^4.4|^5.0"
     },
     "require-dev": {
+        "psr/cache": "^1.0|^2.0|^3.0",
```

Comment:
```suggestion
```

It's fine to rely on `symfony/cache` to provide `psr/cache` for us.

Reply:
Done